requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt

```

Comment:
Is this line intended ?

Reply:
Oh my bad, I just saw [this](https://stackoverflow.com/a/20720019), I didn't know this concept. That's crazy aha